packages/homepage/content/docs/5-secrets.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Secrets"
+authors: ["CompuIves"]
+description: "CodeSandbox allows you to store secrets so you don't share your keys"
+---
+
+## What are secrets?
+
+Secrets are also known as enviroment variables are used to hide sensitive information in your application that you don't want the world to see.
+The most common example for the usage is with API keys.
+
+These secrets will not be transferred between forks and **can only be used in container sandboxes**
+
+## How add secrets in CodeSandbox
+
+You can control the secrets in your server control panel and before adding any it should look like this:
+
+![No Secrets](./images/secrets-1.png)
+
+Let's say you wanna add a google maps api key. You can do it like so:
```

Comment:
```suggestion
Let's say you want to add a Google Maps API key. You can do it like so:
```

Reply:
Yeah, wanna is not a word, I keep forgetting that